python/lsst/daf/base/propertyContainer/propertyList.cc

```diff
@@ -79,32 +93,36 @@ PYBIND11_PLUGIN(_propertyList) {
            when the object just has a trivial default constructor */
         new (&pl) PropertyList();
 
-        static py::object module{PyImport_ImportModule("lsst.daf.base.baseLib"), true};
+        static auto module = py::reinterpret_borrow<py::object>(PyImport_ImportModule("lsst.daf.base.propertyContainer"));
```

Comment:
I think `py::object` variables must not be static - that could cause us to call `Py_DECREF` after the Python interpreter has shut down.